invenio_communities/communities/schema.py

```diff
@@ -240,31 +248,20 @@ def post_dump(self, data, many, **kwargs):
 
         return data
 
-    @pre_load
-    def initialize_custom_fields(self, data, **kwargs):
-        """Ensure custom fields are initialized.
-
-        We need to do that so that validation can take place in case a configured
-        field is marked as required.
-        """
-        data.setdefault("custom_fields", {})
-        return data
-
-    @post_load
-    def lowercase(self, in_data, **kwargs):
-        """Ensure slug is lowercase."""
-        in_data["slug"] = in_data["slug"].lower()
-        return in_data
-
 
 class CommunityParentSchema(BaseCommunitySchema):
     """Community parent schema."""
 
+    id = fields.String(required=True)
+    slug = SanitizedUnicode()
+    metadata = NestedAttribute(CommunityMetadataSchema)
+    access = NestedAttribute(CommunityAccessSchema)
+
 
 class CommunitySchema(BaseCommunitySchema):
     """Community schema."""
 
-    parent = NestedAttribute(CommunityParentSchema, dump_only=True)
+    parent = NestedAttribute(CommunityParentSchema)
```

Comment:
Minor: to be able to set it to None

```suggestion
    parent = NestedAttribute(CommunityParentSchema, allow_none=True)
```